src/index.html

```diff
@@ -1,17 +1,109 @@
 <!doctype html>
 <html lang="en">
+    
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width,user-scalable=no, initial-scale=1.0,maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="/action_page.php" methot="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="formfild" required>
+                    Surname:
+          <input type="text">
+        </div>
+        <div class="formfild">
+                    Name:
+          <input type="text">
+        </div>
```

Comment:
check indents for whole page
```suggestion
        <div class="formfild" required>
          Surname:
          <input type="text">
        </div>
        <div class="formfild">
          Name:
          <input type="text">
        </div>
```